<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Booting window message</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditWindowMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>All power ON , then push button</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditButtonMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Button message</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetIPAddressList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>192.168.0.1</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string> IP address List to deliver destination (Remote control)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddIPAddress">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelIPAddress">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditExecuteBeforeBooter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Execute programs before this booter</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditExecuteAfterBooter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Execute programs after this booter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>650</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK &amp; Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>650</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetBeforeBooter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Program</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Parameter</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidgetAfterBooter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Program</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Parameter</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonAddProgBefore">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelProgBefore">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddProgAfter">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>570</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelProgAfter">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>610</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
